app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,48 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio %>
+  </div>
+
+  <% if @posts_counter.nil? %>
+    <%= f.hidden_field :posts_counter, value: 0 %>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
```

Comment:
- [ ]  Your application is supposed to allow for registering a new user but I was unable to register a new user.

Reply:
Did you click on the confirmation link in your terminal/console?
I've just run `rails db:reset` and tried to register a new user and it's working just fine
![image](https://user-images.githubusercontent.com/13990675/151019604-41e2d33f-5014-48ae-b29b-944b93f0f3a9.png)